service/ziti-tunnel/service/state.go

```diff
@@ -401,6 +403,32 @@ func (t *RuntimeState) UpdateIpv4(ipv4 string) {
 	rts.SaveState()
 }
 
+func UpdateRuntimeStateIpv4(ip string, ipv4Mask int, addDns string) error {
+
+	log.Infof("ip and mask %s %d", ip, ipv4Mask)
```

Comment:
maybe indicating that it's changing?
"updating configuration ip: %s, mask: %d, dns: %t", ip, ipv4Mask, addDns